### PR TITLE
[REF] *: odoo_ui_icons

### PR DESCRIPTION
**Pull request by stefanorigano**
- enterprise: https://github.com/odoo/enterprise/pull/24914

Following new policies in terms of product design-language, this commit
will partially revert 546477d reducing the number of `oi` icons.

It will also improve rendering when icons are rendered at our default
font-size (13px), avoiding blur effects when icons use thin lines.

Since fontAwesome will coexists with `oi` longer than expected, the
commit will also review size utility-classes in order to get consistent
result when used in conjunction with fa's ones (notably `fa-lg`).

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/lib/odoo_ui_icons/style.css [hunk 1]
```diff
@@ -21,321 +20,25 @@
...
+.oi-settings-adjust:before { content: '\e80c'; }
+.oi-apps:before { content: '\e80d'; }
+.oi-panel-right:before { content: '\e810'; }
+.oi-launch:before { content: '\e812'; }
+.oi-studio:before { content: '\e813'; }
+.oi-view-kanban:before { content: '\e814'; }
+.oi-view-cohort:before { content: '\e816'; }
+.oi-view-list:before { content: '\e817'; }
+
+
```
[the remark this answers is not in the document]
Comment (stefanorigano) [reply addons/web/static/lib/odoo_ui_icons/style.css]: fixed, thanks 👍

File: addons/mail/static/src/components/messaging_menu/messaging_menu.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <t t-name="mail.MessagingMenu" owl="1">
         <div class="o_MessagingMenu dropdown" t-att-class="{ 'show': messagingMenu ? messagingMenu.isOpen : false, 'o-mobile': messaging ? messaging.device.isMobile : false }" t-attf-class="{{ className }}" t-ref="root">
             <a class="o_MessagingMenu_toggler dropdown-toggle o-no-caret o-dropdown--narrow" t-att-class="{ 'o-no-notification': messagingMenu ? !messagingMenu.counter : false }" href="#" title="Conversations" role="button" t-att-aria-expanded="messagingMenu and messagingMenu.isOpen ? 'true' : 'false'" aria-haspopup="true" t-on-click="_onClickToggler">
-                <i class="o_MessagingMenu_icon oi oi-large oi-discuss" role="img" aria-label="Messages"/>
+                <i class="o_MessagingMenu_icon fa oi-large fa-comments" role="img" aria-label="Messages"/>
```
Comment (seb-odoo) [inline addons/mail/static/src/components/messaging_menu/messaging_menu.xml:7]: Is it intended to keep `oi-large` with `fa` icons?

Comment (stefanorigano) [reply addons/mail/static/src/components/messaging_menu/messaging_menu.xml:7]: Technically  `oi-large` and `fa-lg` provide the same visual result now.
They have been optimised to get a `16px` icon starting from our `13px` default font-size (before  `fa-lg` returned a `17.33px` icon).

Anyways, `fa-lg` should be used in this case, I agree.

As said, nothing changes visually, but makes sense to use  `fa-lg` when in conjunction with `fa` icons.
Thanks

**Comment by stefanorigano** [comment]
> Hello 👋 The phone on website seems (very) small comparing to 15.2 image image
> 
> Is that something you can fix?

Yes, I'm on it. Thanks @rdeodoo

**Review by adr-odoo — APPROVED** [review]
@robodoo rebase-merge r+

**Pull request closed without merging**